db/migrate/20160615205001_add_paid_at_to_auctions.rb

```diff
@@ -0,0 +1,24 @@
+class AddPaidAtToAuctions < ActiveRecord::Migration
+  def up
+    add_column :auctions, :paid_at, :datetime
+
+    execute <<-SQL
+      UPDATE auctions
+      SET paid_at = CURRENT_TIMESTAMP
```

Comment:
Cool - updated the service class at f23d52e so we can run it via console for all existing auctions and get the payment dates via the CAP API